src/App.js

```diff
@@ -2,35 +2,27 @@ import React, { useEffect } from "react";
 import "./App.css";
 import { request } from "./backend-request";
 import Dashboard from "./components/Dashboard"
+import { AdminDashboard } from "./pages/AdminDashboard"
+import { store } from "./redux/store";
```

Comment:
could you please move store and provider to index.js?

Reply:
ok done!